### PR TITLE
updated to esri-loader 2.14

### PR DESCRIPTION
**Pull request by gavinr**
Also updated to latest `arcgis-js-api` typings. `.otherwise()` was removed in 4.15 so I modified that to `.catch()` per https://developers.arcgis.com/javascript/latest/guide/release-notes/

image

### CODE REVIEW

**Comment by norbertschuler** [comment]
> Also updated to latest `arcgis-js-api` typings. `.otherwise()` was removed in 4.15 so I modified that to `.catch()` per https://developers.arcgis.com/javascript/latest/guide/release-notes/

Took me quite some time today to find out the development branch of our project was broken since esri-loader was updated to version 2.14 and this project is not yet compatible with it. I am looking forward to watching this pull request getting merged.

File: src/ts/components/ArcComposites.tsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ export const WebBase = (props: WebBaseProps) => (
                             }
                         });
                         resolve({ map, view });
-                    }).otherwise((err) => {
+                    }).catch((err) => {
```
Comment (tomwayson) [inline src/ts/components/ArcComposites.tsx:120]: is this a 4.15 thing? If so is the plan to force everyone to 4.15+ starting w/ the next release of react-arcgis?

File: package.json [hunk 2]
```diff
@@ -62,7 +62,7 @@
     ]
   },
   "peerDependencies": {
-    "esri-loader": "^2.10",
+    "esri-loader": "^2.14",
```
Comment (tomwayson) [inline package.json:65]: Unless we are forcing people to 4.15, we should **not** bump `peerDependencies` this should be left as loose as possible.

File: src/ts/components/ArcComposites.tsx (hunk 1 above)
Comment (tomwayson) [reply src/ts/components/ArcComposites.tsx:120]: oops, just read the description

**Comment by lethalbrains** [comment]
@tomwayson 
We are also facing the same issue. I see that pull request has been merged but is there plan to release a version bump? If not, is the recommendation to use the older version?

**Comment by gavinr** [comment]
This was released in v5.1.0. @lethalbrains please test out this new version and let us know. Thanks!

**Pull request merged**